### DISCUSSION TITLE
Please tell me which file needs to be modified to translate the command menus into other languages.

### DISCUSSION THREAD

**Discussion by Yakumo-Yukari** [helix-editor/helix#11977] [Q&A]
![1112](https://github.com/user-attachments/assets/832264d9-a676-4c08-8b68-da524a183e3c)

**Comment by gabydd** [comment]
they start here: https://github.com/helix-editor/helix/blob/38faf74febf3332fb119302324bfd21229d39e14/helix-term/src/commands.rs#L279

**Reply by gabydd** [reply]
typeable commands are here: https://github.com/helix-editor/helix/blob/38faf74febf3332fb119302324bfd21229d39e14/helix-term/src/commands/typed.rs#L2524
